AntistasiOfficial.Altis/CREATE/createSupplyBox.sqf

```diff
@@ -154,6 +154,7 @@ diag_log format ["spawn _fnDeleteMissionIn in 240"];
 
 //Reveal marker when detected
 // TO DO : this do not work
+//wurzel will only work if line below is uncommented
```

Comment:
i saw it working add the beginning but stopped during debugging, i'll try now that the mission is more stable